graylog2-server/src/main/java/org/graylog2/security/hashing/SimpleHashPasswordAlgorithm.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.security.hashing;
+
+import org.apache.shiro.crypto.hash.SimpleHash;
+import org.graylog2.Configuration;
+import org.graylog2.plugin.security.PasswordAlgorithm;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.regex.Pattern;
+
+public class SimpleHashPasswordAlgorithm implements PasswordAlgorithm {
```

Comment:
Could we rename this class to `Sha1PasswordAlgorithm` or `SHA1PasswordAlgorithm`? In the end it's hard coded to SHA-1 and couldn't use any other algorithm.